.github/workflows/ci.yml

```diff
@@ -23,6 +26,10 @@ jobs:
          ghc: ["8.10.7", "9.6.4", "9.8.1"]
          cabal: ["3.10.2.1"]
          os: [ubuntu-latest]
+         variant: [default, no-thunks]
+         exclude:
+           - variant:
+             ${{ github.event_name == 'schedule' ? 'no-thunks' : 'default' }}
```

Comment:
GHA [says](https://github.com/IntersectMBO/ouroboros-consensus/actions/runs/9289759191) that there is a syntax error here, I think line 32 needs more indentation (or it needs to be moved to line 31).

BTW: https://rhysd.github.io/actionlint/ is a nice tool for these kinds of things.

Reply:
Maybe you cannot use variables in a matrix declaration? Even if I indent the line as expected:
![Screenshot 2024-05-30 111104](https://github.com/IntersectMBO/ouroboros-consensus/assets/9791461/44977ecc-210f-460c-b7c1-8c2320c6ec57)


But if I just write a literal there:
![image](https://github.com/IntersectMBO/ouroboros-consensus/assets/9791461/3b3c6666-3aab-42ed-812b-5db437d86b4b)

Reply:
this section was weirdly indented (to 3 spaces?) before I changed it, so I think I'll just fix all the indentation while I'm here

it's the ternary operator that doesn't work, GHA wants a `cond && ifTrue && ifFalse` instead